.github/workflows/lint_code.yml

```diff
@@ -55,8 +55,6 @@ jobs:
         name: Lint JS files
       - run: npm run stylelint
         name: Lint SCSS files
-      - run: npm run prettier
```

Comment:
I would leave this as it is. 
I am using Redmine by JetBrains, and i am running all my Decidim related project in dockerized containers. As a result, i do not have the prettier available on my editor. 

Also, even though I have prettier plugin installed, i still need to do an additional setting in RubyMine, as the plugin read me suggest that "**To run Prettier on save, tick the "Run on save for files" option in Preferences/Settings | Languages \& Frameworks | JavaScript | Prettier.** "

Reply:
The stylelint step will already fail if you have a prettier error, so this step is unnecessary.